kornia/feature/integrated.py

```diff
@@ -42,6 +43,9 @@ def get_laf_descriptors(
     patch_descriptor.eval()
 
     timg: torch.Tensor = img
+    if lafs.shape[1] == 0:
+        warnings.warn(f"LAF contains no keypoints {lafs.shape}, returning empty tensor")
```

Comment:
i'm not very fan of warnings since they pollute running eg tests etc. We could think of using `logging` since it's kind of easy to control the verbosity level. Not sure what are the best practices here. Planning to build a small lib on top of logging+KORNIA_CHECK e.g have more control about the checks and easily get more information when we get bugs and so

Reply:
Sounds good, next pr